crates/noirc_frontend/src/lexer/token.rs

```diff
@@ -4,54 +4,12 @@ use std::{fmt, iter::Map, vec::IntoIter};
 
 use crate::lexer::errors::LexerErrorKind;
 
-impl PartialEq<SpannedToken> for Token {
-    fn eq(&self, other: &SpannedToken) -> bool {
-        self == &other.0.contents
-    }
-}
-impl PartialEq<Token> for SpannedToken {
-    fn eq(&self, other: &Token) -> bool {
-        &self.0.contents == other
-    }
-}
-
-#[derive(Debug, Clone, PartialEq, Eq, Hash)]
-pub struct SpannedToken(Spanned<Token>);
-
-impl From<SpannedToken> for Token {
-    fn from(spt: SpannedToken) -> Self {
-        spt.0.contents
-    }
-}
-
-impl SpannedToken {
-    pub fn new(token: Token, span: Span) -> SpannedToken {
-        SpannedToken(Spanned::from(span, token))
-    }
-    pub fn to_span(&self) -> Span {
-        self.0.span()
-    }
-    pub fn token(&self) -> &Token {
-        &self.0.contents
-    }
-    pub fn into_token(self) -> Token {
-        self.0.contents
-    }
-    pub fn kind(&self) -> TokenKind {
-        self.token().kind()
-    }
-}
-
-impl std::fmt::Display for SpannedToken {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        self.token().fmt(f)
-    }
-}
-
-// XXX(low): Add a Comment Token to force users to have documentation on public functions
-
+/// Represents a token in noir's grammar - a word, number,
+/// or symbol that can be used in noir's syntax. This is the
+/// smallest unit of grammar. A parser may (will) decide to parse
+/// items differently depending on the Tokens present but will
+/// never parse the same ordering of identical tokens differently.
 #[derive(PartialEq, Eq, Hash, Debug, Clone, PartialOrd, Ord)]
-/// All possible tokens allowed in the target language
 pub enum Token {
```

Comment:
Token is now the first item defined in token.rs

Reply:
Why was this doc comment remove?

Reply:
It is not, it was just changed. Github just decided to show this comment on the deleted portion of the diff rather than the addition because it revels in confusion